Source/Scene/Cesium3DTileFeature.js

```diff
@@ -55,6 +53,24 @@ define([
          * @private
          */
         this.primitive = tileset;
+
+        /**
+         * The <code>tile</code> containing this feature.
+         *
+         * @type {Cesium3DTile}
+         *
+         * @private
+         */
+        this.tile = content._tile;
```

Comment:
I don't think it is worth it since this can be accessed through `content` and I think was only used in the tests.